### ISSUE TITLE
Admin Dash

### ISSUE THREAD

**Issue by brianseminara** [Sank6/Discord-Bot-List#307]
Is there a way for us to see the unverified bots page on the website before we accept/deny it?

**Comment by Sank6** [comment]
`/bots/:id`

**Comment by wolfier74** [comment]
Yes, admin and bot checker dashboard for verifing, removing, deleting and editing bots is great idea. 👍
